src/python/pants/core/util_rules/archive.py

```diff
@@ -51,6 +52,7 @@ async def maybe_extract(extractable: MaybeExtractable) -> ExtractedDigest:
                 argv=("/bin/bash", "-c", f"{extraction_cmd_str}"),
                 input_digest=digest,
                 description=f"Extract {snapshot.files[0]}",
+                level=LogLevel.DEBUG,
```

Comment:
Extracting an archive doesn't seem substantial enough to persist as a log to a user, imo. It will still show up in the dynamic UI.